_pages/user-experience/sign-in-sign-out.md

```diff
@@ -45,19 +45,19 @@ Don't use an icon instead of a text label. Buttons may include the Login.gov log
 
 <ul class="usa-icon-list padding-top-2">
   <li class="usa-icon-list__item">
-    {% include check_icon.html content=do_button_label style="text-green" %}
+    {% include icon_list.html icon_name="check_circle" content=do_button_label style="text-success" %}
   </li>
   <li class="usa-icon-list__item">
-    {% include check_icon.html content=do_state_styling style="text-green" %}
+    {% include icon_list.html icon_name="check_circle" content=do_state_styling style="text-success" %}
   </li>
   <li class="usa-icon-list__item">
-    {% include check_icon.html content=do_font_styling style="text-green" %}
+    {% include icon_list.html icon_name="check_circle" content=do_font_styling style="text-success" %}
   </li>
   <li class="usa-icon-list__item">
-    {% include dont_icon.html content=dont_external_link %}
+    {% include icon_list.html icon_name="cancel" content=dont_external_link style="text-red" %}
```

Comment:
Given where we  landed with https://github.com/18F/identity-design-system/pull/405, we may want to get ahead and replace this with `error` instead (and throughout)

```suggestion
    {% include icon_list.html icon_name="cancel" content=dont_external_link style="text-error" %}
```

Reply:
yes! thanks.

Reply:
fixed in https://github.com/18F/identity-dev-docs/pull/449/commits/066047311beb4a0c0ad059c9c8fd340238a3ab97